src/main/resources/templates/db_scripts/schema.json

```diff
@@ -928,7 +928,7 @@
     {
       "run": "after",
       "snippetPath": "oaipmh/createOaiPmhViewFunction.sql",
-      "fromModuleVersion": "19.2.5"
+      "fromModuleVersion": "19.3.0"
```

Comment:
Given that 19.2.5 didn't trigger the execution of this script. Should this be 19.3.1 rather than 19.3.0?

If 19.2.5 being less than 19.3.0 is what caused the script not to execute, then could 19.3.0 being less than 19.3.1 cause it not to execute this time?

@julianladisch Maybe you can help me understand how this works again, because I'm not sure I'm following it.

Reply:
Yes, it should be 19.3.1 to enforce running the .sql file when upgrading from any version below 19.3.1.